source/test/pgg_mode_test/PGGHostPGGSymbiontUnitTest.test.cc

```diff
@@ -232,17 +235,18 @@ TEST_CASE("PGGHost DistribResources", "[pgg]") {
         double sym_portion = 0;
 
         double sym_points = sym_portion;
-        double host_points = host_portion * num_syms; // * by num_syms bc points are added during each iteration through host's syms
+        double host_points = host_portion * num_syms; // * by num_syms bc points are added during each iteration through host'symbiont syms

```

Comment:
there are more of these that I didn't mark so make sure to check for everywhere it says 'symbiont'